src/app/cli/interpreter.cpp

```diff
@@ -197,7 +197,9 @@ const std::map<std::string, Interpreter::Evaluator> &Interpreter::mEvaluatorMap
 };
 
 const std::map<std::string, std::string> &Interpreter::mUsageMap = *new std::map<std::string, std::string>{
-    {"config", "config get pskc\n"
+    {"config", "config get accesscode\n"
+               "config set accesscode <9-digits-thread-access-passcode>\n"
```

Comment:
Just come to realize that if the access code has the empty spaces removed, then the string is effectively a epskc. So probably we should change this command to "config set epskc"?

Reply:
Offline chat with @wgtdkp, fine to keep it consistent with the terminology with Thread Specification.